private/rules/maven_publish.bzl

```diff
@@ -5,13 +5,15 @@ MavenPublishInfo = provider(
         "javadocs": "Javadoc jar file for documentation files",
         "artifact_jar": "Jar with the code and metadata for execution",
         "source_jar": "Jar with the source code for review",
+        "extra_classifiers": "Extra classifiers for Artifacts",
+        "extra_artifacts": "Extra artifacts such as tars, wars",
     },
 )
 
 _TEMPLATE = """#!/usr/bin/env bash
 
 echo "Uploading {coordinates} to {maven_repo}"
-{uploader} {maven_repo} {gpg_sign} {user} {password} {coordinates} {pom} {artifact_jar} {source_jar} {javadoc}
+{uploader} {maven_repo} {gpg_sign} {user} {password} {coordinates} {pom} {artifact_jar} {source_jar} {javadoc} {extra_classifiers} {extra_artifacts}
```

Comment:
Can we pair up classifier and artifact? It'll make scanning the command line when debugging a little simpler. Something like `classifier=path` would be easy to iterate over, and it's really clear what would go with what. It would also mean that we could use a `for` loop to read the additional artifacts.

Reply:
i can make this change but this is not inline with attr change